pkg/cmd/roachtest/tests/admission_control_disk_bandwidth_overload.go

```diff
@@ -71,10 +68,12 @@ func registerDiskBandwidthOverload(r registry.Registry) {
 
 			roachtestutil.SetAdmissionControl(ctx, t, c, true)
 
-			dataDir := "/mnt/data1"
-			if err := setBandwidthLimit(ctx, t, c, c.CRDBNodes(), "wbps", 128<<20 /* 128MiB */, false, dataDir); err != nil {
-				t.Fatal(err)
-			}
+			const provisionedBandwidth = 128 << 20 // 128 MiB
+			t.Status(fmt.Sprintf("limiting disk bandwidth to %d bytes/s", provisionedBandwidth))
+			staller := roachtestutil.MakeCgroupDiskStaller(t, c,
+				false /* readsToo */, false /* logsToo */)
+			staller.Setup(ctx)
```

Comment:
nb: cgroupDiskStaller.Setup does nothing if logsToo is false, but that's probably okay and worth calling anyway to satisfy the interface expectations. Just to be safe you could move the staller instantiation and setup call to before `c.Start`.

Reply:
Thanks for the callout! I am going to leave it as is for `admission-control/snapshot-overload-io` since it only initializes the staller for the bandwidth subtest, and it does all the other bandwidth setup in the same if-block. I think it is ok to not need to do it before `c.Start`.